merkle/objhasher/objhasher.go

```diff
@@ -17,26 +17,26 @@ package objhasher
 
 import (
 	"github.com/benlaurie/objecthash/go/objecthash"
-	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/merkle/hashers"
 )
 
 type objmaphasher struct {
-	merkle.MapHasher
+	hashers.MapHasher
 }
 
 type objloghasher struct {
-	merkle.LogHasher
+	hashers.LogHasher
 }
 
-// NewMapHasher returns a new ObjectHasher based on the passed in MapHasher
-func NewMapHasher(baseHasher merkle.MapHasher) merkle.MapHasher {
+// GetMapHasher returns a new ObjectHasher based on the passed in MapHasher
+func GetMapHasher(baseHasher hashers.MapHasher) hashers.MapHasher {
```

Comment:
nit: I think this one was better named as NewMapHasher.

Same below (NewLogHasher).